frontstage/notifications/notifications.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from urllib import parse as urlparse
+from urllib.error import HTTPError
+
+from flask import current_app
+import requests
+from structlog import wrap_logger
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+class AlertViaGovNotify:
+    """Notify Api handler"""
+
+    @staticmethod
+    def send(email, name):
+        notification = {
+            "emailAddress": email,
+            "name": name
+        }
+        if bool(email.strip()):
+            url = urlparse.urljoin(current_app.config['RM_NOTIFY_GATEWAY_URL'],
+                                   current_app.config['NOTIFICATION_TEMPLATE_ID'])
+
+            response = requests.post(url, auth=current_app.config['BASIC_AUTH'],
+                                     timeout=current_app.config['REQUESTS_POST_TIMEOUT'],
+                                     json=notification)
+            if response.status_code != 201:
+                logger.error(
```

Comment:
This log is not covered in any of the tests?

Reply:
why would you test a log ?

Reply:
You should have a test that covers the response not being a 201 though

Reply:
ehm I donto think so. You test what actually change the state of an object/data , you do nto test if you are having a void condition and logging it. The rei snot business function or error hadler. Is litteraly not implemented.